Source/Charts/Renderers/HorizontalBarChartRenderer.swift

```diff
@@ -265,15 +265,23 @@ open class HorizontalBarChartRenderer: BarChartRenderer
                 context.setFillColor(dataSet.color(atIndex: j).cgColor)
             }
 
-            if !dataProvider.barRoundingCorners.isEmpty {
-                let cornerRadius = barRect.size.height / 2
-                let bezierPath = UIBezierPath(roundedRect: barRect,
-                                              byRoundingCorners: dataProvider.barRoundingCorners,
-                                              cornerRadii: CGSize(width: cornerRadius,
-                                                                  height: cornerRadius))
+            if dataProvider.shouldRoundBars {
```

Comment:
Is it possible to use `drawBar(context: CGContext, barRect: CGRect, rounded: Bool)` that was added in `BarChartRenderer` here?